.zappr.yaml

```diff
@@ -0,0 +1 @@
+X-Zalando-Type: tools
```

Comment:
It is partially a tool (for tech radar visualization) and document (public tech radar releases).
I would prefer 'document' type, thx.